docs/guides/nextjs-setup.md

```diff
@@ -0,0 +1,285 @@
+# Next.js Setup with Eppo Feature Flags
+
+This guide will walk through how to setup an Eppo Feature Flag on a client and server rendered component in a Next.js app. More specifically, this guide will walk through the set up and settings of a sample Next.js app.
+
+## Start a new Next.js React App
+
+### Create the Next.js app
+
+```tsx
+npx create-next-app
+```
+Provide the following answers when prompted:
+- What is your project named? `react-sdk-walkthrough`
+- Would you like to use TypeScript? `Yes`
+- Would you like to use ESLint? `Yes`
+- Would you like to use Tailwind CSS? `No`
+- Would you like to use `src/` directory? `No`
+- Would you like to use App Router? (recommended) `Yes`
+- Would you like to customize the default import alias (@/*)? `No`
+
+### Run the Next.js app
+
+```tsx
+cd react-sdk-walkthrough
+yarn dev
+```
+
+Open a browser to the location output for `local` (e.g., [http://localhost:3000](http://localhost:3000/))
+
+## Eppo flag setup
+1. [Create and copy an SDK key](/sdks/sdk-keys.md) in your Eppo app if you don't already have one.
+2. [Create a feature flag](/quick-starts/feature-flag-quickstart.md/#2-create-a-flag) in your Eppo app.
+3. Make sure your flag is turned on the environment that correlates to your SDK key.
+
+## Client rendered component setup
+
+### Install the Eppo JS client SDK
+
+```bash
+yarn add @eppo/js-client-sdk
+```
+
+### Create a randomization provider
+
+1. Create a provider based off our [Usage in React](/sdks/client-sdks/javascript#usage-in-react) documentation
```

Comment:
This is confusing: do I need to go to that page? It seems to do the same as what's listed below

Reply:
Good call, going to make that an intro vs a step in the process